cloudsync_onedrive.py

```diff
@@ -1106,17 +1109,19 @@ def rename(self, oid, path):  # pylint: disable=too-many-locals, too-many-branch
     def _parse_time(time_str):
         try:
             if time_str:
-                ret_val = arrow.get(time_str).timestamp
+                ret_val = arrow.get(time_str).int_timestamp
             else:
                 ret_val = 0
         except Exception as e:  # pragma: no cover
             log.error("could not convert time string '%s' to timestamp: %s", time_str, e)
             ret_val = 0
         return ret_val
 
-    def _make_path_relative_to_shared_folder_if_needed(self, path):
+    def _make_path_relative_to_shared_folder_if_needed(self, path, force=False):
```

Comment:
If I'm reading this correctly, `force=True` will permit this method to return None instead of the original path, when the path is not relative to the shared folder, correct?